app/views/semesters/clones/_form.html.haml

```diff
@@ -8,7 +8,8 @@
     = hidden_field_tag :model_id, @model.id
   .field
     = f.label :beginning
-    %br
+    %p.instructions
+      Typically the end date of the previous semester
```

Comment:
Rock on. We _could_ require this, but it's possible that would cause problems recovering from an error state of some sort, and I don't think the risk is justified. Since we default the beginning to the end of the previous, there's less chance for a mistake